### PR TITLE
Describing parameters

### PR DESCRIPTION
**Pull request by Lujkn**
I have added a description for parameters of flatten_grid

### CODE REVIEW

File: include/deal.II/grid/grid_generator.h [hunk 1]
```diff
@@ -1535,6 +1535,11 @@ namespace GridGenerator
    * parallel::distributed::Triangulation, as well as when the input
    * Triangulation contains hanging nodes.
    *
+   * @param[in] in_tria is the base input for a new flat triangulation.
+   * @param[out] out_tria the desired flattened triangulation constructed from the in_tria.
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
   * @param[out] out_tria The desired flattened triangulation constructed from @p in_tria.
```

File: include/deal.II/grid/grid_generator.h [hunk 2]
```diff
@@ -1535,6 +1535,11 @@ namespace GridGenerator
    * parallel::distributed::Triangulation, as well as when the input
    * Triangulation contains hanging nodes.
    *
+   * @param[in] in_tria is the base input for a new flat triangulation.
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]:
```suggestion
   * @param[in] in_tria The input for a new flat triangulation.
```

File: include/deal.II/grid/grid_generator.h [hunk 3]
```diff
@@ -1535,6 +1535,11 @@ namespace GridGenerator
...
+   * @param[out] out_tria the desired flattened triangulation constructed from the in_tria.
+   * be built to consist of the cells that correspond to the (selected portion
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: I think you are missing the first part of this sentence. Could you please add the text?

**Comment by masterleinad** [comment]
Also, please run the indentation script at `contrib/utilities/indent`. Let us know if you need help with that.

File: include/deal.II/grid/grid_generator.h [hunk 4]
```diff
@@ -1702,7 +1707,7 @@ namespace GridGenerator
                                         const double,
                                         const double,
                                         const unsigned int,
-                                        const bool);
+                                        const bool);
```
Comment (drwells) [inline include/deal.II/grid/grid_generator.h]: Please remove all trailing spaces.

File: doc/news/changes/minor/20190708LuelEmishaw [hunk 5]
```diff
@@ -0,0 +1 @@
+I described [In] and [out] parameters of flatten_triangulation()
```
Comment (bangerth) [inline doc/news/changes/minor/20190708LuelEmishaw]: Take a look at the other files in this directory. They're all pretty formulaic in their format. See if you can imitate it.

**Comment by bangerth** [comment]
Yes, nice!

**Pull request merged**